Filter/FilterInterface.php

```diff
@@ -28,7 +28,7 @@
      * @param ProxyQueryInterface $queryBuilder
      * @param string              $alias
      * @param string              $field
-     * @param string              $value
+     * @param string[]            $value
```

Comment:
It's probably more accurate than `string` indeed, but I'm not sure all the values in the array are strings. I'd rather use `array` or `mixed` or even `mixed[]` here

Reply:
In the case of a date range filter for instance, you will get an array, not a string

Reply:
Fixed